src/js/services/lang.js

```diff
@@ -78,40 +80,30 @@ const languageRequest = () => {
       }
     });
   });
-};
-
-const submitLanguage = () => {
-  const submitBtn = $('#submit-btn');
-  const input = $('#lang-req')[0];
 
-  if (input.value.length > 0) {
-    // If user input present, adjust UI and add event listener to submit data on click to amplitude
-    submitBtn.css({ opacity: '100%', cursor: 'pointer' });
-    submitBtn.on('click', () => {
-      clearInterval(checkLangInput);
-      langForm.css('pointer-events', 'none');
+  // Toggle submit button UI and button state based on user input
+  langForm.on('keyup', () => {
+    submitBtn.removeAttr('disabled');
+    if (input.value == '') {
```

Comment:
seems like you can use `this.value` instead and no need for keeping an input variable in that case 😊

Reply:
`this.value` is coming back as undefined, but we could use `langForm.val()`?